CHANGELOG.md

```diff
@@ -5,6 +5,28 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v3.41.0](https://github.com/buildkite/agent/tree/v3.42.0) (2023-01-05)
+[Full Changelog](https://github.com/buildkite/agent/compare/v3.41.0...v3.42.0)
+
+ ### Added
+ - Add an in-built hierarchical status page [#1873](https://github.com/buildkite/agent/pull/1873) (@DrJosh9000)
+ - Add an `agent-startup` hook that fires at the same time as the `agent-shutdown` hook is registered [#1778](https://github.com/buildkite/agent/pull/1778) (@donalmacc)
+
+ ### Changed
+- Use `stdlib` errors instead of `github.com/pkg/errors` [#1872](https://github.com/buildkite/agent/pull/1872) (@DrJosh9000)
```

Comment:
- Feels a bit weird to put stdlib in code quotes when it's not a specific package / module / ... `errors` is a package though.
- It's also an internal cleanup, so is it worth mentioning at all?